src/gatherers/html-without-javascript.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/* Note that this returns the outerHTML of the <body> element, not the documentElement. */
+
+const HTML = require('./html');
+
+class HTMLWithoutJavaScript extends HTML {
+  get name() {
+    return 'htmlWithoutJavaScript';
+  }
+
+  afterThirdReloadPageLoad(options) {
+    const driver = options.driver;
+
+    this.artifact = {};
+    return driver.sendCommand('DOM.getDocument')
```

Comment:
Can you add some comments in here for the various steps, so it's clearer what's going on and what you're choosing to do?